tools/pylint-extensions/pylint-guidelines-checker/pylint_guidelines_checker.py

```diff
@@ -357,6 +356,10 @@ class ClientMethodsHaveTracingDecorators(BaseChecker):
         ),
     )
     ignore_clients = ["PipelineClient", "AsyncPipelineClient", "ARMPipelineClient", "AsyncARMPipelineClient"]
+    ignore_functions = ["send_request"]
+    ignore_decorators = ["typing.overload", "builtins.classmethod", "azure.core.tracing.decorator.distributed_trace", 
+        "azure.core.tracing.decorator_async.distributed_trace_async"]
+    ignore_internal = False
```

Comment:
Is it not possible to simply annotate the relevant azure-core things with suppression annotations instead of special-casing these to never flag in the first place?